syntax = "proto3";

import "google/protobuf/empty.proto";

option go_package = "gen/go/cards";

service CardsService {
  rpc UploadCardsData (CardsUploadRequest) returns (CardsUploadResponse);
  rpc CleanCardsTable (google.protobuf.Empty) returns (CardsCleanResponse);
}

message CardsUploadRequest {
  string file_path = 1;
  int32 month = 2;
  int32 year = 3;
}

message CardsUploadResponse {
  string status = 1;
}

message CardsCleanResponse {
  string status = 1;
}
